todo_app/data/trello_items.py

```diff
@@ -0,0 +1,97 @@
+from flask import session
+from datetime import datetime
+import requests
+import os
+
+def trello_get_items():
+    Items=[]
```

Comment:
Should be `items` rather than `Items`.

The Python convention is PascalCase for class names, not variables.